.github/workflows/release.yml

```diff
@@ -0,0 +1,32 @@
+name: Release
+
+on:
+  push:
+    tags:
+      - 'v*'
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v4
+
+    - name: Configure CMake
+      run: |
+        cmake -S . -B build -DCMAKE_BUILD_TYPE=Release
+
+    - name: Build source package
+      run: cmake --build build --target ggml_package
+
+    - name: Create Release
+      uses: softprops/action-gh-release@v2
```

Comment:
It's better to avoid using external actions in release jobs since there is a security risk. We already have a forked `ggml-org/action-create-release@v1`, used in llama.cpp.